makefile

```diff
@@ -0,0 +1,45 @@
+default:
+	cat Makefile
+
+get_texts:
+	@echo "Getting books..."
+	/bin/bash get_the_books.sh
+
+raven_line_count: 
+	@echo "Line count in The Raven"
+	@cat pg17192.txt | wc -l 
+
+raven_word_count:
+	@echo "Word count in The Raven"
+	@cat pg17192.txt | wc -w
+
+raven_counts: 
+	@echo "Lowercase 'raven' count:"
+	@cat pg17192.txt | grep 'raven' | wc -l
+	@echo "Titlecase 'Raven' count:"
+	@cat pg17192.txt | grep 'Raven' | wc -l 
+	@echo "Case-insensitive 'raven' count:"
+	@cat pg17192.txt | grep -i 'raven' | wc -l
+
+total_lines:
+	@cat *.txt | wc -l
+
+total_words:
+	@cat *.txt | wc -w
+
+setup: 
+	python3 -m venv env
+	./env/bin/pip install --upgrade pip 
+	./env/bin/pip install -r requirements.txt
+
+lint:
+	@echo "Running linting..."
+	./env/bin/pylint src tests
+
+test: lint
+	@echo "Running non-integration tests..."
+	./en/bin/pytest -v -m "not integration"
```

Comment:
oops, should be `env/bin`, you just missed the `v`

Reply:
Kudos for using the pytest in the environment!

However, the `-v -m "not integration"` if I'm not mistaken could be followed by a `tests/` just to be on the safe side and not have pytest go trying to run tests outside of that directory.